rust/sbp/src/messages/gnss.rs

```diff
@@ -488,4 +505,38 @@ pub mod sv_id {
             }
         }
     }
+
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+    pub enum Constellation {
+        /// GPS
+        Gps = 0,
```

Comment:
```suggestion
        GPS = 0,
```

Reply:
Hmm, that is part of the autogenerated code. I'll look into why it changed capitalization. Otherwise, next time anyone makes the target, it'll be overwritten.

Reply:
There is already a rule in place that turns GPS into Gps, so for consistency, I've added a similar rule for BDS and GAL.